notebooks/225-stable-diffusion-text-to-image/225-stable-diffusion-text-to-image.ipynb

```diff
@@ -37,7 +59,8 @@
     ">To use this diffusion model, you must be a registered user in 🤗 Hugging Face Hub. You will need to use an access token for the code below to run. For more information on access tokens, refer to [this section of the documentation](https://huggingface.co/docs/hub/security-tokens).\n",
     ">You can login on Hugging Face Hub in notebook environment, using following code:\n",
     "```python\n",
-    "## login to huggingfacehub to get access to pretrained model \n",
+    "<a id=\"2\"></a>\n",
+    "## login to huggingfacehub to get access to pretrained model [&#8657;](#0)\n",
```

Comment:
it is optional step, no need to add it to table of content (it can mislead uses)

Reply:
and it is code snippet at all :)